docs/pages/installation.mdx

```diff
@@ -107,21 +107,192 @@ information on obtaining Teleport binaries compatible with Teleport Cloud.
+| `(=teleport.latest_ent_docker_image=)-fips` | Yes | [Ubuntu 20.04](https://hub.docker.com/\_/ubuntu) |
 
-We provide pre-built `amd64`, `arm`, and `arm64` Docker images for every version of Teleport Enterprise.
+For testing, we always recommend that you use the latest release version of
+Teleport Enterprise, which is currently `(=teleport.latest_ent_docker_image=)`.
 
-(!docs/pages/includes/enterprise/docker-images.mdx!)
+[Ubuntu 20.04](https://hub.docker.com/\_/ubuntu)-based images for non-FIPS 
+Teleport are available from our [Legacy Amazon ECR Public repository](https://gallery.ecr.aws/gravitational/teleport-ent).
 
 </TabItem>
 </Tabs>
 
+### Running Teleport on Docker
+
+When running a container from one of the images listed above, consider the
+container equivalent to running the `teleport` binary. The Teleport container
+requires access to a file system and network ports.
+
+Our [Helm charts](#helm) take care of the Teleport container's system
+requirements for you if deploying Teleport on Kubernetes, so these instructions
+are relevant only when running a standalone Teleport container in some other
+environment.
+
+#### Configuration
+
+Teleport processes read their configuration from a local file path, which is
+`/etc/teleport.yaml` by default. Make sure this file path is mounted to your
+Teleport container.
+
+#### Data directory
+
+All Teleport processes read from and write to a data directory, which by default
+is `/var/lib/teleport`. Make sure the data directory is mounted to your Teleport
+container.
+
+#### License file
+
+If your Teleport Enterprise container runs the Auth Service, you will need to
+give it access to a license file at the path named in the configuration, which
+is `/var/lib/teleport/license.pem` by default. Make sure a license exists at
+this location in the Teleport container's data directory.
+
+#### Other file paths
+
+Depending on the configuration settings you assign on your Teleport container,
+you will need to make sure that any file paths you name are mounted on the
+container. 
+
+For example, if you are running the Teleport Proxy Service on a container, you
+need to mount this directory to your Teleport container, then assign the
+following fields in the container's configuration file to the appropriate paths:
+
+```yaml
+proxy_service:
+  https_keypairs:
+  - key_file: /my/path/key.pem
+    cert_file: /my/path/cert.pem
+```
+
+See the Teleport [Configuration Reference](reference/config.mdx) for whether a
+field you would like to assign requires a file path.
+
+#### Ports
+
+A single Teleport process can run multiple services, each of which listens on a
+specific set of ports depending on your configuration. See our [Networking
+Reference](reference/networking.mdx#ports) for the ports on your Teleport
+container to expose.
+
+### Example of running a Teleport container
```

Comment:
Would have the OSS and ent versions.  With a note that cloud would just support other services.

Reply:
I'm not sure we need to cover all editions here. In this new iteration, the "Example..." section is an illustration of tying together all of the instructions from other subsections in the "Docker" section to show a user how to execute `docker run` with a Teleport container. I don't want to make this section a canonical "Getting Started with Docker" guide since the demo you get is so limited.

I've suggested adding more context to the "Example..." section to make the scope clearer to users.

Reply:
Ok thanks, that's good